client/blocks/upload-image/index.jsx

```diff
@@ -246,6 +253,19 @@ class UploadImage extends Component {
 		this.setState( { uploadedImage: null } );
 	};
 
+	componentWillMount() {
+		// use defaultImage as uploadedImage if set.
+		const { defaultImage } = this.props;
+		if ( defaultImage ) {
+			this.setState( {
+				uploadedImage: defaultImage,
```

Comment:
When component initial state depends on props, one usually needs to also implement `componentWillReceiveProps`. The `defaultImage` prop can be updated from outside (i.e., when rerendering the form with another data). In this case, the `state.uploadedImage` will have the old value and weird bugs will happen.